experimental/trusted_shuffler/backend/src/main.rs

```diff
@@ -36,12 +39,27 @@ pub struct Opt {
     listen_address: String,
 }
 
+lazy_static! {
+    // We use START_TIME to time the time elapsed between every request and response.
+    static ref START_TIME: Instant = Instant::now();
+}
+
 async fn handler(request: Request<Body>) -> Result<Response<Body>, hyper::Error> {
     match (request.method(), request.uri().path()) {
         (&Method::POST, "/request") => {
             let body = hyper::body::to_bytes(request.into_body()).await?;
-            info!("Received request: {:?}", body);
-            // Send back the response body.
+
+            let request_start = START_TIME.elapsed();
+            // Currently we assume the backend takes no time, i.e., no time elapsed between request
+            // and response. For consistency with client and server we give the time
+            // twice.
+            log::info!(
+                "backend,{:?},{},{}",
+                body,
+                request_start.as_millis(),
+                request_start.as_millis()
```

Comment:
Why does it print `request_start` 2 times?

Reply:
Is part of the problem that server and client logs are being collected in the same "file"? It may be useful to define the CSV schema upfront, so that we know whether we are collecting the appropriate data from the right components. In other words: what do we want to plot eventually?